tests/tests_api/test_personal_info.py

```diff
@@ -57,8 +57,8 @@ def test_personal_info_check_existence_returns_400_with_existing_info(api_client
             "last_name": existing_personal_info.last_name,
         },
     )
-    assert response.status_code == status.HTTP_400_BAD_REQUEST
```

Comment:
I think it makes sense to also add a test with a Cyrillic email, making sure that `status.HTTP_400_BAD_REQUEST` is returned.

BTW, I noticed that my RFC-something-compliant regex lets the user enter an email address with one character for domain zone (e.g. "mail@mail.**c**"), but Django rejects it as an invalid email. Just a side note.

Reply:
Added test